doc/en/source/k8sIntegration.rst

```diff
@@ -0,0 +1,186 @@
+**********************
+Kubernetes Integration
+**********************
+
+The components of FogFlow can be built via source code as well as in docker environment using docker-compose tool. In docker environment each component of FogFlow is running on a single container. Whole FogFlow system will have to re-start in case any single component container goes down and if any single service is overloaded it cannot scale to handle the load.  
+To overcome these issues FogFlow would be migrated to Kubernetes. FogFlow components will be setup in Kubernetes cluster environment based on end user requirement. Kubernetes cluster can be deployed in various configurations:
```

Comment:
change from "**To overcome these issues FogFlow would be migrated to Kubernetes**" to To overcome these issues FogFlow has migrated to Kubernetes

And 
from "**FogFlow components will be setup in Kubernetes cluster environment based on end user requirement**." to FogFlow components will be deployed in Kubernetes cluster environment based on end user requirement. 

From **"Kubernetes cluster can be deployed in various configurations"** to Various cluster configuration can be deployed: